numpy/core/src/npysort/timsort.c.src

```diff
@@ -2121,46 +2131,54 @@ npy_force_collapse(char *arr, run *stack, npy_intp *stack_ptr,
 int
 npy_timsort(void *start, npy_intp num, void *varr)
 {
-    size_t len = PyArray_ITEMSIZE(varr);
-    PyArray_CompareFunc *cmp = PyArray_DESCR(varr)->f->compare;
+    PyArrayObject *arr = varr;
+    size_t len = PyArray_ITEMSIZE(arr);
```

Comment:
Maybe name it `elsize` so that the definitions match with the above one? The check can maybe be `elsize` for both for similarity.

But you did not change it, so I guess that is probably already part of your "style cleanup".

Reply:
Yeah, I will probably change it later, but there are so many style nits that I want to clean up that they would just get in the way of a review, so I decided to make that a separate, lower priority, task.

Reply:
I may actually do two style cleanups, one just for code style, and another doing a refactor. An instance of the latter would be using `npy_intp` rather than `size_t`.